recipe/meta.yaml

```
@@ -27,7 +27,7 @@ requirements:
     - typing-extensions >=3.7.4
   run:
     - colorama
-    - meson >=0.60.0
+    - meson >=0.62.0
```

```suggestion
    - meson >=0.60.0
```

Looks like only the meson-python build requirements are updated to meson [`>=0.62.0`](https://github.com/FFY00/meson-python/blob/601fe26bc27ffce6b7ab3ba3b7e3d17b22029c77/pyproject.toml#L5), run requirements are still at [`>=0.60.0`](https://github.com/FFY00/meson-python/blob/601fe26bc27ffce6b7ab3ba3b7e3d17b22029c77/pyproject.toml#L29)

---

Good catch, I opened https://github.com/FFY00/meson-python/pull/116 for that